corehq/apps/sms/migrations/0034_auto_20191007_0756_noop.py

```diff
@@ -0,0 +1,56 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
this is a noop migration (no sql get's run) but Django still wants the model changes recorded.